### ISSUE TITLE
Use Exposure Notifications API

### ISSUE THREAD

**Issue by dtchanpura** [nic-delhi/AarogyaSetu_Android#385]
Recently for contact tracing, both Google and Apple joined together to create an API which helps contact tracing respecting individual’s Privacy. To avoid using Location we can try to implement this new Exposure Notifications API.

References:
Documentation: https://www.google.com/covid19/exposurenotifications/
Sample Implementation: https://github.com/google/exposure-notifications-android

**Comment by legendavey** [comment]
Sadly this app will not be eligible, as it uses location tracking *in addition* to Bluetooth. It also uses a central server that is the opposite of the decentralised model envisioned by Apple and Google.

**Comment by RohitY24** [comment]
It would be awesome if the app could utilize the Exposure notifications, it would allow for system-wide tracking which will be more accurate.

**Comment by g-savitha** [comment]
Yes, health apps of other countries are already utilising this new feature.
